Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -338,6 +339,13 @@ void ProjectData::parseProcesses(BaseLib::ConfigTree const& processes_config,
                         "dimension");
             }
         }
+        else if (type == "DensityDrivenFlow")
```

Comment:
It might be better to make the process name more specific. In general, density driven flow occurs due to solute concentration gradient or temperature gradient. So far this process targets only the temperature gradient.